server/migrations/20200608192629-add-discussion-drafts.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('DiscussionDrafts', {
+      id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
+      author_id: { type: DataTypes.INTEGER, allowNull: false, references: { model: 'Addresses', key: 'id' } },
+      title: { type: DataTypes.TEXT, allowNull: true },
+      body: { type: DataTypes.TEXT, allowNull: true },
+      tag: { type: DataTypes.STRING, allowNull: true },
+      chain: { type: DataTypes.STRING, allowNull: true },
```

Comment:
No chain association?

Reply:
Okay, I dunno why we don't have references on the other models for `chain` either...probably makes sense to revisit later